lib/content-services/src/lib/category/services/category-tree-datasource.service.ts

```diff
@@ -76,7 +77,10 @@ export class CategoryTreeDatasourceService extends TreeService<CategoryNode>  {
                 );
             }),
                 toArray(),
-                map(res =>  ({entries: res, pagination: pagingResult.list.pagination})));
+                map(res =>  {
+                    const sortedResults = res.sort((category1, category2) => category1.nodeName.localeCompare(category2.nodeName));
```

Comment:
I don't think it is proper way to do that. We should do sorting on backend and just proper request to backend on frontend. In other case we just sort single 'page'. We have raised backend jira already for that so this should be done after that backend jira is finished: 
https://alfresco.atlassian.net/browse/ACS-4664

Reply:
Oh, I was not aware that there was backend JIRA already raised for this task. In that case, I'll close this PR, and we'll raise a new one once that task is complete. I've also linked the 2 JIRAs together. Thanks!

Reply:
ok, thank you! :)